### ISSUE TITLE
mobileview: cant switch tab between `code` and `preview` tab

### ISSUE THREAD

**Issue by jaiakash** [processing/p5.js-web-editor#3228]
### Increasing Access

Currently, the p5.js web editor does not allow users to view or switch between the `code` and `preview` tabs while a project is running. This feature is essential for users to review and modify code while simultaneously observing the output.

While exploring examples on the p5.js website, I observed that on the desktop view, switching between `code` and `preview` tabs is possible, allowing for simultaneous code review and output observation. However, this functionality is not available on mobile devices, where switching tabs are disabled while the project is running. This limitation impairs the ability to debug and iterate on code when using mobile devices efficiently.

Problem

https://github.com/user-attachments/assets/8d0b466f-d59f-44e5-b93d-d8075db07e64

### Feature enhancement details

Add a tab to view `code` tab when it is running, in mobile view.

<img src="https://github.com/user-attachments/assets/1158b0c8-4c70-47a3-8d82-378602adbf25" alt="PHOTO-2024-09-07-13-49-21" width="300"/>

**Comment by dianaodias** [comment]
I agree, this would be a great feature to increase accessibility to the editor for those that don't have access to computers.

**Comment by priyaanshusoni** [comment]
amazing

**Comment by akkarn1689** [comment]
Hi @jaiakash,

I would like to contribute to this issue. Could you please assign it to me?

**Comment by lakshay-jainn** [comment]
@raclim @Jatin24062005  hi !. I have added a Solution to this problem in my linked PR #3346 . can you please check it out and tell is there any areas of improvement? . Thank you
